### PR TITLE
Expand nightmare snippets

### PR DESCRIPTION
**Pull request by Stone94**
#### Summary
Content "Adds several new nightmares"

#### Purpose of change

I felt like contributing something, and I felt the nightmares could use some variety and _flavor_.

#### Describe the solution
I wrote some short story nightmares without writing a novel.

#### Describe alternatives you've considered
none

#### Testing
ran it through a json validator since i'm unsure how to force nightmares to occur.

### CODE REVIEW

File: data/json/snippets/nightmares.json [hunk 1]
```diff
@@ -8,8 +8,20 @@
...
+      "You try to get comfortable.  You hear a loud skittering sound but see nothing.  The sound grows louder,  and yet you see nothing.  Desperate,  you search for the source of the sound,  you find nothing except for an itch in your ear.  You reach to scratch it,  causing hundreds of tiny spiders burst out,  you find the source of the sound.  You wake up.",
+      "You open your eyes to a blinding light, surrounded by medical equipment,  and a group of panicking individuals dressed in blue scrubs.  They seem to be doctors.  Your vision is blurry from the anesthesia,  but one of them seems to be holding a mass of flesh;  it appears to be alive.  You glance down and find your chest has been cut open.  The doctors start screaming as the mass of flesh rips them apart.  You are next…  You wake up."
       "You awaken from a nightmare about the deaths of your loved ones.",
-      "You awaken from a nightmare about the Cataclysm not having happened and being stuck in an office job."
+      "You awaken from a nightmare about the Cataclysm not having happened and being stuck in an office job.
```
Comment (Maleclypse) [inline data/json/snippets/nightmares.json:24]:
```suggestion
      "You awaken from a nightmare about the Cataclysm not having happened and being stuck in an office job."
```

**Comment by Maleclypse** [comment]
You need to lint the file.  Copy the whole file into the tab on this website https://dev.narc.ro/cataclysm/format.html hit lint then copy it back into your json file over the original and save it.

File: data/json/snippets/nightmares.json [hunk 2]
```diff
@@ -8,6 +8,18 @@
...
+      "You awaken feeling hollow;  you dreamt of your friends and family,  only to realize your new reality is the nightmare.",
+      "You abruptly awake,  relief washes over you as you were no longer naked in the street,  and the zombies weren't laughing at you.",
+      "You feel something gently embrace you as you sleep.  You open your eyes,  but it's too dark to see,  you struggle to move,  but it has your arms and legs in a cocoon of darkness,  it has stolen your mouth,  and you cannot scream.  It starts to devour you whole,  starting with your feet,  unable to escape,  and unable to scream.  You take your last breaths.  Then you wake up,  it was just a dream.",
+      "Your in a meadow,  unsure of how you got here.  The sun is shining,  the birds are singing,  and children are playing;  all is peaceful.  So why do the hairs on your neck stand at attention?  Something is not quite right about this peace.  You turn around,  and the children have you surrounded.  The faces of the children have vanished,  replaced by a shark-like grin and beady black eyes.  They reach out with their pale,  claw-like hands.  However,  you cannot move,  you cannot escape…  you awake.",
```
Comment (NetSysFire) [inline data/json/snippets/nightmares.json:17]:
```suggestion
      "You are in a meadow,  unsure of how you got here.  The sun is shining,  the birds are singing,  and children are playing;  all is peaceful.  So why do the hairs on your neck stand at attention?  Something is not quite right about this peace.  You turn around,  and the children have you surrounded.  The faces of the children have vanished,  replaced by a shark-like grin and beady black eyes.  They reach out with their pale,  claw-like hands.  However,  you cannot move,  you cannot escape…  you awake.",
```

File: data/json/snippets/nightmares.json [hunk 3]
```diff
@@ -8,6 +8,18 @@
...
+      "You open your eyes,  and you are not where you slept.  You have been strapped to a chair in a dimly lit room with damp,  rusty walls.  A clown spattered in blood approaches you from behind,  wielding a rusty drill,  dancing around you until he meets you face to face.  He toys with the trigger,  enjoying the sound.  You plead for him to let you go,  he replies with this maniacal laugh.  He inches the roaring drill towards your face,  playing with you,  for you are his toy.  You can't help but to watch in horror as the drill bit slowly inches its way towards your eye,  unable to look away,  unable to close your eyes.  This is it.  You wake up,  safe and sound.",
```
Comment (NetSysFire) [inline data/json/snippets/nightmares.json:18]:
```suggestion
      "You open your eyes,  and you are not where you slept.  You have been strapped to a chair in a dimly lit room with damp,  rusty walls.  A clown covered in blood splatters approaches you from behind,  wielding a rusty drill,  dancing around you until he meets you face to face.  He toys with the trigger,  enjoying the sound.  You plead for him to let you go,  he replies with this maniacal laugh.  He inches the roaring drill towards your face,  playing with you,  for you are his toy.  You can't help but to watch in horror as the drill bit slowly inches its way towards your eye,  unable to look away,  unable to close your eyes.  This is it.  You wake up,  safe and sound.",
```

Minor reword so it sounds better, there was also a typo.
Also, imagine if the player gets this nightmare, waking up "safe and sound", only to get attacked by something. Nothing critical but yet another funny message.

File: data/json/snippets/nightmares.json [hunk 4]
```diff
@@ -8,6 +8,18 @@
...
+      "You regain consciousness and realize you are at the bottom of a deep pit.  You can make out a bucket just opposite of you.  Within the bucket you find a bottle of lotion.  You jolt awake and shudder.",
+      "You feel as if your being watched.  Several feet away from you are a pair of burning yellow eyes.  A long, slimy appendage snakes it's way towards you,  hauling you towards an abyssal pit of serrated teeth.  You wake up.",
```
Comment (NetSysFire) [inline data/json/snippets/nightmares.json:20]:
```suggestion
      "You feel as if you are being watched.  Several feet away from you are a pair of burning yellow eyes.  A long, slimy appendage snakes it's way towards you,  hauling you towards an abyssal pit of serrated teeth.  You wake up.",
```

**Review by NetSysFire — COMMENTED** [review]
All of the "You wake up" at the end of the sentences might lead to message duplication which should be addressed because it looks odd.

"Blah blah nightmare. You wake up.
You start walking. You wake up."

File: data/json/snippets/nightmares.json (hunk 2 above)
Comment (Stone94) [reply data/json/snippets/nightmares.json:17]: I didn't think about the possible double messaging, I'll fix it after work.

**Comment by Stone94** [comment]
> You need to lint the file. Copy the whole file into the tab on this website https://dev.narc.ro/cataclysm/format.html hit lint then copy it back into your json file over the original and save it.

Ok ill do that as soon as I can.

**Comment by Tykato** [comment]
There's a typo "You take your last breaths.  it was just a dream."

'it' should be 'It'.

**Pull request merged**